index1.js

```diff
@@ -0,0 +1,67 @@
+class AwesomeBooks {
+  booksList = [];
+
+  addToBooksList = (title, author) => {
+    const book = {
+      tit: title,
+      aut: author,
+    };
+    this.booksList.push(book);
+  };
```

Comment:
- [OPTIONAL] I would recommend to you that you should try to implement some validation measures, so that the users would not be able to submit empty book objects.
- [ ]  Also, try to clear your input fields at every submit to avoid a lot of duplication of the same book objects. Take a look at what I am saying.

[your-page.webm](https://user-images.githubusercontent.com/26597432/204539446-d07df909-8765-46eb-b772-323b8f2f6258.webm)